react/features/settings/functions.js

```diff
@@ -118,6 +118,7 @@ export function getMoreTabProps(stateful: Object | Function) {
  */
 export function getProfileTabProps(stateful: Object | Function) {
     const state = toState(stateful);
+    const { user: { name: walletName = '' } = {} } = state['features/base/jwt'];
```

Comment:
thanks @Chima1707
I also done that [https://github.com/aeternity/jitsi-meet/pull/35/commits/ed20159d811594acc0503346f3531a8216926d37] to make it everywhere the same